.changeset/shaggy-donkeys-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'firebase': patch
+---
+
+Update `/compat/app` type path to accommodate the new `"moduleResolution": "bundler"` mode of TypeScript 5.0+.
```

Comment:
This is too specific, this is not a special accommodation for TypeScript 5, it was just wrong in the first place. Simply "Fixed the types path for `compat/app`" should work.

If you want to make sure this is linked back to the original bug report, you can put "Fixes [link to bug]" in the PR description. This will close the issue once you merge this, and it will appear as ("fixes #xxxx") in the Version Packages PR during release which will make it easy for the release operator to follow the link and understand the background.

Reply:
Sounds good! I'll update the changeset